docs.html

```diff
@@ -5,6 +5,11 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Just a suggestion, you can use your info in meta tags just to give a touch of your personal project.
For eg, <meta name="author" content="your name" />

Reply:
Ok will try to implement this from further projects.